taipy/core/exceptions/exceptions.py

```diff
@@ -18,7 +17,9 @@ class ConfigCoreVersionMismatched(Exception):
 
     def __init__(self, config_core_version: str, core_version: str) -> None:
         self.message = (
-            f"Core version {config_core_version} in Config does not match with version of Taipy Core {core_version}."
+            f"The version {config_core_version} of Core's entities does not match version of Taipy Core {core_version}."
+            f" Please update the core entities to be compatible with Taipy Core {core_version}"
+            " using the `taipy migrate ...` command. For more information, please run `taipy help migrate`"
```

Comment:
Please let me know if the message if clear enough

Reply:
Neat.